tcf_website/static/common/sortClasses.js

```diff
@@ -5,6 +5,7 @@ let sortNumberAsc = 1;
 let sortRatingAsc = -1;
 let sortDifficultyAsc = 1;
 let sortGpaAsc = -1;
+let sortRecencyAsc = -1;
```

Comment:
Kinda out of scope for your PR because you were following the existing pattern, but do you know of any reason why these can't be bools? lol

Reply:
We might've talked about this earlier, but I think we used this in cmpByProp() to make directional sorting easier.